xbmc/GUIInfoManager.cpp

```diff
@@ -2400,6 +2407,7 @@ const infomap container_bools[] ={{ "onnext",           CONTAINER_MOVE_NEXT },
                                   { "numpages",         CONTAINER_NUM_PAGES },
                                   { "numitems",         CONTAINER_NUM_ITEMS },
                                   { "actualitems",      CONTAINER_ACTUAL_ITEMS },
+                                  { "nonfolderitems",   CONTAINER_NONFOLDER_ITEMS },
                                   { "currentpage",      CONTAINER_CURRENT_PAGE },
```

Comment:
Because not every non-folder item is a file item, imo. What about channels games, epg events? Are those file items? I don't think so.